lib/core/ui/speaker_image.dart

```diff
@@ -23,8 +21,8 @@ class SpeakerImage extends StatelessWidget {
         child: DecoratedBox(
           decoration: BoxDecoration(
             image: DecorationImage(
-              image: AssetImage('${ImageAsset(
-                speakerImage ?? 'dashatar.png',
+              image: AssetImage('${const ImageAsset(
+                'dashatar.png',
```

Comment:
Bu kısımı geri alarak şu şekilde düzenleyebiliriz:
```suggestion
                speakerImage,
```
Ayrıca `speakerImage `null olabileceği için parametre olarak `'dashatar.png'` gönderebiliriz.

Reply:
Before, we did not have images of all speakers. We need to support speakers without photos.

For that reason, `speakerImage` should be nullable and optional. If it is null, then we need to show default dashatar image.